tensorboard/webapp/theme/_tb_palette.scss

```diff
@@ -45,15 +49,19 @@ $tf-orange: (
   300: mat-color($mat-orange, 500),
   400: mat-color($mat-orange, 500),
   500: mat-color($mat-orange, 500),
+  600: mat-color($mat-orange, 700),
```

Comment:
For the tf-slate, 600 was matching 500 yet, here, it matches 700. Is this intended?

Reply:
No, it was not intentional. Now 600 matches 700 in both tf-slate and tf-orange.